hieradata/common.yaml

```diff
@@ -1210,6 +1250,9 @@ router::nginx::robotstxt: |
   User-agent: deepcrawl
   Disallow: /
 
+sidekiq::host: 'redis-1.backend'
```

Comment:
I've spoken to @alexmuller and agreed we should remove the namespace so as not to cause confusion that this could be related to a Sidekiq module. This follows the convention set nu `app_domain`, `deployable_applications` etc. See c179cbbdab20247d10b1663c78b00183119de9fc